modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1451,6 +1452,86 @@ definition {
 		AppBuilderAdmin.validateItemNameOnTable(itemName = "Content of form deployed as widget.");
 	}
 
+	@description = "Reuse my object's existing fields when creating a new form view"
```

Comment:
Hey @nkkwon , thanks for the script!

The purpose of this validation is to check if we can reuse the existing fields from a form view to another one. So, if you are setting a field as required, it would be good to see if when adding this field to a new form, this property is persisted.

You can check some properties like, help text, options, predefined value, etc.